src/core/Accordion/Accordion.stories.tsx

```diff
@@ -204,3 +209,18 @@ export const StaticAndFullyOpen = {
     },
   },
 };
+
+export const CustomClickableButton = {
```

Comment:
I know we had this earlier but I think "WithCustomOnClick" would be better here as it describes what the story is portraying - i.e. an accordion with a custom onClick action